src/towncrier/build.py

```diff
@@ -69,12 +68,9 @@ def _main(
             project_date,
             answer_yes,
         )
-    except Exception as e:
-        if isinstance(e, ConfigError):
-            print(e)
-        else:
-            traceback.print_exc(file=sys.stderr)
-        raise
+    except ConfigError as e:
+        print(e, file=sys.stderr)
+        sys.exit(1)
```

Comment:
This seems like the simplest change that 1) avoids the traceback for `ConfigError`s and 2) doesn't double up the traceback on regular exceptions.  Though, admittedly, 2) is actual scope creep for this PR.  `:]`  I tried your suggestion which resulted in test failures due to getting `SystemExit` rather than `ValueError` exceptions, for example.  If you think that is in fact better, I can switch back and update the tests.